Resources/Lang/en_US.json

```diff
@@ -688,7 +688,7 @@
   "NinjaInfoLong": "(Impostors):\nAs the Ninja, you can use your kill button to Mark a target (single click) or kill normally (double click). You may then Shapeshift to teleport to the Marked target and kill them.",
   "AnonymousInfoLong": "(Impostors):\nAs the Anonymous, you can Shapeshift to force your target to report whoever you killed this round.\nIf you killed nobody that round, the target will report their own dead body as if they had died.\nNote: This does not work on Lazy nor Lazy Guy, and this ability will work regardless of whether the body can normally be reported.",
   "MinerInfoLong": "(Impostors):\nAs the Miner, you can shapeshift to teleport back to the last vent you were in.",
-  "KillingMachineInfoLong": "(Impostors):\nAs the Killing Machine, you have a very short kill cooldown with tiny vision. However, you cannot vent, sabotage, report, and call emergency meetings.\n\nNote: You will bypass any shields, killing bait and beartrap won't take any effect",
+  "KillingMachineInfoLong": "(Impostors):\nAs the Killing Machine, you have a very short kill cooldown with tiny vision. However, you cannot vent, sabotage, report, or call emergency meetings.\n\nNote: You will bypass any shields, killing bait and beartrap won't take any effect",
```

Comment:
it's nor

Reply:
"nor" would be incorrect word choice. "or" a better conjunction to use here.

Reply:
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/21760bc7-c476-4c7d-888a-52367f14011f)
idk what to tell u

Reply:
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/98eecb21-604d-410e-bf63-41aba6ae244e)
it even has it in the definiton

Reply:
either one can be used here. but if you want to use "nor" we can.